src/cache/cache.jl

```diff
@@ -208,13 +208,13 @@ function compute_coriolis(ᶜcoord, ᶠcoord, params)
         global_geom = Spaces.global_geometry(axes(ᶜcoord))
         if global_geom isa Geometry.DeepSphericalGlobalGeometry
             @info "using deep atmosphere"
-            coriolis_deep(coord::Geometry.LatLongZPoint) = Geometry.LocalVector(
-                Geometry.Cartesian123Vector(zero(Ω), zero(Ω), 2 * Ω),
-                global_geom,
-                coord,
+            coriolis_deep(coord::Geometry.LatLongZPoint) = Geometry.UVWVector(
+                zero(Ω),
+                2 * Ω * cosd(coord.lat),
+                2 * Ω * sind(coord.lat),
```

Comment:
LGTM ; 
Vector norms are comparable within machine precision (checks in F32 ClimaAtmos) ;

```
Main:
norm(ᶠf¹²)
0.000119080156f0
norm(ᶜf³)
8.4202184f-5
```

```
PR: 
norm(ᶠf¹²)
0.000119080156f0
norm(ᶜf³)
8.4202184f-5
```

Reply:
We decided not to make the change as it is preferred to have Cartesian3Vector here, but there is a missing ClimaCore method needed for the conversion.